src/NzbDrone.Core/Parser/Parser.cs

```diff
@@ -455,7 +455,7 @@ public static class Parser
         private static readonly Regex LanguageRegex = new Regex(@"(?:\W|_)(?<italian>\b(?:ita|italian)\b)|(?<german>german\b|videomann)|(?<flemish>flemish)|(?<greek>greek)|(?<french>(?:\W|_)(?:FR|VOSTFR)(?:\W|_))|(?<russian>\brus\b)|(?<dutch>nl\W?subs?)|(?<hungarian>\b(?:HUNDUB|HUN)\b)|(?<spanish>\b(?:español|castellano)\b)",
                                                                 RegexOptions.IgnoreCase | RegexOptions.Compiled);
 
-        private static readonly Regex YearInTitleRegex = new Regex(@"^(?<title>.+?)(?:\W|_)?(?<year>\d{4})",
+        private static readonly Regex YearInTitleRegex = new Regex(@"^(?<title>.+?)(?:\W|_)*?(?<year>\d{4})",
```

Comment:
This regex will also trim any trailing non-word characters from the series name, which would affect something like `American Dad! (2005) S01E01`, though I don't think any releases actually include the exclamation mark.
Still, it might be safer to explicitly check for whitespace instead: `^(?<title>.+?)\s*(?:\W|_)?(?<year>\d{4})`

Thoughts, @markus101?

Reply:
Generally they don't, but could probably do something like `^(?<title>.+?)[-_. ]+?(?<year>\d{4})` which would only capture it on characters that are actually used as separators (we've converted a bunch of `\W|_` to those 4 characters in other regex). You want to use `+` instead of `*` because we want one or more, instead of zero or more.

Reply:
I used `*` to handle a case like `Series2022` but I guess that's considered invalid, right?

Reply:
I think it’s safe to treat that as invalid and let it fail.